docs/SimpleFormIterator.md

```diff
@@ -70,6 +70,7 @@ const OrderEdit = () => (
 | Prop     | Required | Type           | Default               | Description                         |
 |----------|----------|----------------|-----------------------|-------------------------------------|
 | `addButton` | Optional | `ReactElement` | - | Component to render for the add button |
+| `clearButton` | Optional | `boolean` | `true` | When `false`, the user cannot clear the array |
```

Comment:
If it's a simple boolean, then the prop name isn't right. 

In general, we prefer to define boolean props that take the value `true` when used, so:

```jsx
<SimpleFormIterator disableClear>
```

is preferable to:

```jsx
<SimpleFormIterator clearButton={false}>
```

In addition, we already have other `disableXXX` props for this component, so for consistency this prop should follow the same pattern.

Reply:
updated : b987e628a3c25ea157aaa022f81781f4055abf5c